src/content/docs/en/core-concepts/astro-components.mdx

```diff
@@ -248,8 +248,40 @@ import Wrapper from '../components/Wrapper.astro';
 </Wrapper>
 ```
 
-
+:::tip
 Use a `slot="my-slot"` attribute on the child element that you want to pass through to a matching `<slot name="my-slot" />` placeholder in your component.
+:::
+
+To pass multiple HTML elements into a component's `<slot/>` placeholder without a wrapping `<div>`, use the `slot=""` attribute on [Astro's `<Fragment/>` component](/en/reference/api-reference/#fragment-):
+
+```astro title="src/components/CustomTable.astro" "<slot name="header"/>" "<slot name="body"/>"
+---
+// Create a custom table with named slot placeholders for head and body content
+---
+<table class="bg-white">
+  <thead class="sticky top-0 bg-white"><slot name="header"/></thead>
+  <tbody class="[&_td:nth-child(odd)]:bg-gray-100"><slot name="body"/></tbody>
+</table>
+```
+
+Inject multiple rows and columns of HTML content using a `slot=""` attribute to specify the `"header"` and `"body"` content. Individual HTML elements can also be styled:
+
+```astro title="src/components/my-table.astro" {5-7, 9-13} '<Fragment slot="header">' '<Fragment slot="body">'
+---
+import CustomTable from './CustomTable.astro';
+---
+<CustomTable>
+  <Fragment slot="header"> <!-- pass table header -->
+    <tr><th>Column 1</th><th>Column 2</th></tr>
+  </Fragment>
+
+  <Fragment slot="body"> <!-- pass table body -->
```

Comment:
```suggestion
  <Fragment slot="header"> <!-- passed into table header -->
    <tr><th>Column 1</th><th>Column 2</th></tr>
  </Fragment>

  <Fragment slot="body"> <!-- passed into table body -->
```

I don't know if this makes sense, but it does in my head. For clarity - the slots terminology doesn't seem to have a clear _directionality_ to it, in my opinion (`name` <--> `slot` attributes).

Reply:
So, my attempt at this has been to use the phrase "slot placeholder" when talking about the `<slot />` element itself (that happens to have a `name` attribute, but I'm not emphasizing this). Then, when talking about the content that will go into the slot, I *do* say "slot attribute" because in this case, the slot is in fact an attribute, not an element. 

So, I'm trying to less equate attributes and more distinguish which role "slot" has in each case... hopefully somewhat successfully?  :sweat_smile: